src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/IntendedDataUseReportButton.java

```diff
@@ -18,33 +22,31 @@ public class IntendedDataUseReportButton implements IsWidget {
 	public Button button;
 	public IsACTMemberAsyncHandler isACTMemberAsyncHandler;
 	AccessRequirement ar;
-	BigPromptModalView copyTextModal;
-	IntendedDataUseGenerator iduGenerator;
+	DialogView dialog;
+	IntendedDataUseReportWidget iduReportWidget;
+	SynapseAlert synAlert;
 	
 	@Inject
-	public IntendedDataUseReportButton(Button button, IsACTMemberAsyncHandler isACTMemberAsyncHandler, BigPromptModalView copyTextModal, IntendedDataUseGenerator iduGenerator) {
+	public IntendedDataUseReportButton(Button button, IsACTMemberAsyncHandler isACTMemberAsyncHandler, DialogView dialog, DivView divView, SynapseAlert synAlert, IntendedDataUseReportWidget iduReportWidget) {
 		this.button = button;
 		this.isACTMemberAsyncHandler = isACTMemberAsyncHandler;
-		this.iduGenerator = iduGenerator;
-		this.copyTextModal = copyTextModal;
-		copyTextModal.addStyleToModal("modal-fullscreen");
-		copyTextModal.setTextAreaHeight("450px");
+		this.dialog = dialog;
+		this.iduReportWidget = iduReportWidget;
+		this.synAlert = synAlert;
+		divView.add(synAlert);
+		divView.add(iduReportWidget);
+		dialog.configure(IDU_MODAL_TITLE, divView, null, "Cancel", null, true);
+		dialog.addStyleName("modal-fullscreen");
 		button.setVisible(false);
 		button.addStyleName("margin-left-10");
-		CallbackP<String> mdCallback = md -> {
-			showIDUs(md);
-		};
 		button.addClickHandler(event -> {
-			iduGenerator.gatherAllSubmissions(ar.getId().toString(), mdCallback);
+			iduReportWidget.configure(ar.getId().toString());
+			dialog.show();
 		});
 	}
 
-	public void showIDUs(String md) {
-		copyTextModal.configure(IDU_MODAL_TITLE, IDU_MODAL_FIELD_NAME, md.toString());
-		copyTextModal.show();
-	}
-
 	public void configure(AccessRequirement ar) {
+		synAlert.showError("Please use the dynamic IDU Report wiki widget instead of this snapshot. ${iduReport?accessRestrictionId="+ar.getId()+"}");
```

Comment:
ACT only UI.  If an ACT member ever clicks this, point them to the wiki widget.  When the ACT Dashboard is released, my hope is that we can remove the IntendedDataUseReportButton altogether

Reply:
I don't think we have a place for that anywhere in the new dashboard (yet). Clicking an AR ID takes you to the old AR page.